lib/matplotlib/axes/_axes.py

```diff
@@ -798,15 +798,15 @@ def axhline(self, y=0, xmin=0, xmax=1, **kwargs):
 
         Returns
         -------
-        line : :class:`~matplotlib.lines.Line2D`
+        line : a `.Line2D` instance
```

Comment:
Was this a rebase error? I'm not sure it was supposed to change. (And same in a few other places below.)

Reply:
Thanks for the review! Actually, this one and the changes in other places are done on purpose: to unify and simplify the `Line2D` code reference.

Could you let me know what's the concern about these changes? I checked the built docs, it seems fine to me:

https://23447-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/_as_gen/matplotlib.axes.Axes.axhline.html#matplotlib-axes-axes-axhline

Reply:
The part after the colon is the type, so adding "a instance" is somewhat redundant. Now there is no description on these, but in every other place with a description (and probably some without), the "a instance" bit was removed because it's wasting precious space (though space is not an issue here.)

Reply:
```suggestion
        line : `~matplotlib.lines.Line2D`
```

I know that some other places still have the "instance" thing. But that should be removed there as well. We also wouldn't write `s : a str instance`.

See also https://matplotlib.org/devel/documenting_mpl.html#referencing-types.